src/roll20/page-script.js

```diff
@@ -28,6 +28,23 @@ function updateHP(name, current, total, temp) {
     }
 }
 
+function updateCombatTracker(combat) {
+    const index = combat.findIndex(x => x[2]);
+    if (index === -1) {
+        console.warn("It's apparently nobody's turn :/");
+    } else {
+        const c = combat.splice(index, combat.length);
+        combat.splice(0, 0, ...c);
```

Comment:
Nice, so you're splicing the combatant whose turn it is and anyone after them and placing them at the beginning. I'd just add a comment to explain it 
`// Re-order list with the combatant whose turn it is at the top of the list, as that's how Roll20 needs it displayed`